components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -355,6 +359,27 @@ def validate_logs_dir(self):
         except ValueError as ex:
             raise ValueError(f"logs_directory is invalid: {ex}")
 
+    def load_execution_container_name(self):
+        if self.execution_container is not None:
+            # Accept configured value in clp-config.yml for debug purposes
+            return
+
+        with open(self.os_release_file_path) as os_release_file:
+            parsed = {}
+            for line in os_release_file:
+                var, val = line.strip().split("=")
```

Comment:
technically we can add a guard to ensure split does return two items, but since the os release file should have a relatively fixed format, this code should be fine

Reply:
Right, I was thinking about adding  `try ... except` blocks to handle such errors / FileNotFoundError, but since users are not expected to modify this `<clp-package>/etc/os-release` file, such handlings could be redundant. In any case, users will get notified by the Exception stack.